src/components/Card/PokemonCard.tsx

```diff
@@ -1,15 +1,14 @@
 import { useNavigate } from "react-router-dom";
 import { PokiType } from "../../utils/Types";
 import { LazyLoadImage } from "react-lazy-load-image-component";
-// import "react-lazy-load-image-component/src/effects/blur.css";
-
+import { getComplementaryColor } from "../../utils/typesColor";
 type cardProps = {
   id: number;
   name: string;
   imgURL: string;
   baseColor?: string;
   ability?: string;
-  type: PokiType;
+  type?: PokiType;
 };
 
 function PokemonCard({
```

Comment:
Use 

```
 const PokemonCard: React.FC<PokemonCardProps> = () => 
```
Follow this convention for other components as well.

Reply:
I forgot to change it here